CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED]
 
+### Changed
```

Comment:
@cjao header here should be "Docs" instead of "Changed"

Reply:
In earlier changelog entries, documentation-related comments went under the "Added" or "Changed" headers:

E.g. 
```
## [0.18.7] - 2022-01-24                                                                                                                                                                                                                       
                                                                                                                                                                                                                                               
### Changed                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                               
- Many stylistic improvements to documentation, README, and CONTRIBUTING.
```
Just want to confirm that we should use the "Docs" header this time.